enos/modules/vault_verify_removed_node/scripts/verify_raft_remove_peer.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+function fail() {
+  echo "$1" 1>&2
+  exit 1
+}
+
+[[ -z "$RETRY_INTERVAL" ]] && fail "RETRY_INTERVAL env variable has not been set"
+[[ -z "$TIMEOUT_SECONDS" ]] && fail "TIMEOUT_SECONDS env variable has not been set"
+[[ -z "$VAULT_ADDR" ]] && fail "VAULT_ADDR env variable has not been set"
+[[ -z "$VAULT_TOKEN" ]] && fail "VAULT_TOKEN env variable has not been set"
+
+binpath=${VAULT_INSTALL_DIR}/vault
+test -x "$binpath" || fail "unable to locate vault binary at $binpath"
+
+getSysHealth() {
+  $binpath read -format=json sys/health sealedcode=299 haunhealthycode=299 removedcode=299 | jq '.removed_from_cluster'
+}
+
+getStatus() {
+  $binpath status --format=json | jq '.removed_from_cluster'
```

Comment:
Since we rely on the exit status of these we should have `jq` return non-zero (`-e`). I'd probably use `jq -eMc` for the fastest and most compliant response.